build/crds/build-crds.sh

```diff
@@ -39,10 +40,20 @@ copy_ob_obc_crds() {
 }
 
 generating_crds_v1() {
-  echo "Generating v1 in crds.yaml"
+  echo "Generating ceph crds"
   "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/ceph.rook.io/v1" output:crd:artifacts:config="$OLM_CATALOG_DIR"
   # the csv upgrade is failing on the volumeClaimTemplate.metadata.annotations.crushDeviceClass unless we preserve the annotations as an unknown field
   $YQ_BIN_PATH w -i cluster/olm/ceph/deploy/crds/ceph.rook.io_cephclusters.yaml spec.versions[0].schema.openAPIV3Schema.properties.spec.properties.storage.properties.storageClassDeviceSets.items.properties.volumeClaimTemplates.items.properties.metadata.properties.annotations.x-kubernetes-preserve-unknown-fields true
+
+  echo "Generating cassandra crds"
+  "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/cassandra.rook.io/v1alpha1" output:crd:artifacts:config="$CASSANDRA_CRDS_DIR"
+  # Format with yq for consistent whitespace
+  $YQ_BIN_PATH read $CASSANDRA_CRDS_DIR/cassandra.rook.io_clusters.yaml > $CASSANDRA_CRDS_DIR/crds.yaml
+  rm -f $CASSANDRA_CRDS_DIR/cassandra.rook.io_clusters.yaml
+
+  echo "Generating nfs crds"
+  "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/nfs.rook.io/v1alpha1" output:crd:artifacts:config="$NFS_CRDS_DIR"
```

Comment:
No `yq` read?

Reply:
There is a `yq read` on the next line, is that what you mean?

Reply:
Oh wow I must have missed it. Never mind!